public/components/register_model/__tests__/register_model_tags.test.tsx

```diff
@@ -208,4 +203,92 @@ describe('<RegisterModel /> Tags', () => {
       })
     );
   });
+
+  it('should allow adding one more tag when registering new version if model group has only two tags', async () => {
```

Comment:
Q: where is the two tags of the model group specified?

Reply:
For now, the tags data of register model and register are from `useModelTags` hook. There should be refactor to model group related data for register version after tags API integration. The keys in register version should from the model group detail which called in the parent component. Since we use [model partials](https://github.com/opensearch-project/ml-commons-dashboards/pull/142/files#diff-9883fda44728e3bc9b30bb2084d4e719a7e450c060aec02daa0a0548a179c8a4R70) to display all the model sections, we need to find a way to pass these data from outside.